### DISCUSSION TITLE
Root owned helix.log file and dir

### DISCUSSION THREAD

**Discussion by alper** [helix-editor/helix#6428] [General]
I can't find an issue for it but after I setup my system cleanly, I ended up with a `~/.cache/helix/helix.log` file structure that was entirely owned by `root`.

That shouldn't be the case and I deleted the entire thing promptly because it was blocking some other apps that wanted to do something in `~/.cache`.

I installed using Homebrew. Could that be it?

**Comment by cd-a** [comment]
This was a known issue on homebrew for linux 3 years ago, but was promptly fixed. 

Did you run `sudo brew install` by any chance?
